web/app/js/components/DeploymentDetail.jsx

```diff
@@ -185,7 +185,7 @@ export default class Deployment extends React.Component {
             }
             <TabbedMetricsTable
               resource="pod"
-              entity={this.state.deploy}
+              Resource={this.state.deploy}
```

Comment:
same comment as above

Reply:
yeppp, bad find/replace

Reply:
yes, i like that!

Reply:
the above change didn't seem up update with your last commit? Or is it a weird caching issue on my side?

The props still seem to be:
```
               resource="pod"
               Resource={this.state.deploy}
``` 
in the `TabbedMetricsTable` component on line 187/188